CIScripts/Build/BuildFunctions.ps1

```diff
@@ -178,6 +178,7 @@ function Invoke-ExtensionBuild {
     $vRouterMSI = "$vRouterRoot\extension\vRouter.msi"
     $vRouterCert = "$vRouterRoot\extension\vRouter.cer"
     $utilsMSI = "$vRouterRoot\utils\utils.msi"
+    $vTestSrcPath = "vrouter\utils\vtest\"
```

Comment:
K, if we're going with this, then we should rename $vRouterRoot to $vRouterBuildRoot or something. 

And then move $vTestSrcPath 2 lines below (so it's not mixed with build output paths). $vTestPath should also be renamed to $vTestBuilPath.

And maybe we should instead have $vTestSrcRoot and then two other vars:
$vTestScenariosPath (for xmls) and $vTestRunnerPath (for powershell script).